code-challenges-401/arrayReverse/README.md

```diff
@@ -0,0 +1,9 @@
+# Reverse an Array 
+In this code challenge I created an array and returned the array in reversed order.
+
+## Challenge
+The code challenge asked me to create a function which takes in an array as an argument. Without utilizing any built-in methods available to JavaScript, I was asked to return an array with elements in reversed order.
+
+## Approach and Efficiency 
+I took the pproach of understanding what the challenge was asking me to do in simplistic terms. I then began to think of different JavaScript code I could utilize without using methods.
```

Comment:
That's a really great way to approach a problem! I would suggest looking at this problem again, checking on Google because there is a way of doing it without using .push!